.travis.yml

```diff
@@ -200,6 +202,7 @@ script:
   - ./configure $CONFIGURE_FLAGS
   ## We run `make check` because that's what https://jenkins.torproject.org does.
   - if [[ "$DISTCHECK" == "" ]]; then make check; fi
```

Comment:
I just noticed this job takes significantly longer to run, so maybe we could skip the `make check` when we're doing `make test-stem`? We'd probably want to do something like `make all && make test-stem` though to avoid parallelization problems.

Reply:
+1

I wish we could fix the parallelisation problems, but that's a separate issue that deserves a separate ticket. (And they're so hard to debug.)

I suggest make src/app/tor, unless test-stem uses some of the other build products.

Reply:
Did these things and build takes less time now.